src/_check.sh

```diff
@@ -120,3 +121,29 @@ function _check_base_required_packages() {
   fi
   return $status
 }
+
+function check_etl_status() {
+  title "Checking Couchbase-to-Postgres ETL status"
+
+  RAW_OUTPUT=$(compose_client exec plextracapi npm run pg:etl:status)
+
+  # Find the json content by looking for the first line that starts
+  # with an opening brace and the first line that starts with a closing brace.
+  JSON_OUTPUT=$(echo "$RAW_OUTPUT" | sed '/^{/,/^}/!d')
+
+  # Find the summary content by finding the first line that starts
+  # with a closing brace and selecting all remaining lines after that one.
+  SUMMARY_OUTPUT=$(echo "$RAW_OUTPUT" | sed '1,/^}/d')
+
+  ETLS_COMBINED_STATUS=$(echo $JSON_OUTPUT | jq -r .etlsCombinedStatus)
+
+  if [[ "$ETLS_COMBINED_STATUS" == "HEALTHY" ]] 
+  then
+    info "All ETLs are in a healthy status."
+  else
+    error "One or more ETLs are in an unhealthy status."
+  fi
+
+  msg "$SUMMARY_OUTPUT\n"
+  debug "$JSON_OUTPUT\n"
+}
```

Comment:
```suggestion
  title "Checking Couchbase-to-Postgres ETL status"

  etlStatusOut=$(compose_client exec plextracapi npm run pg:etl:status 2>&1) || true # don't break if the script exits unhealthy

  # Find the json content by looking for the first line that starts
  # with an opening brace and the first line that starts with a closing brace.
  etlStatusJson=$(echo "$etlStatusOut" | sed '/^{/,/^}/!d')

  # Find the summary content by finding the first line that starts
  # with a closing brace and selecting all remaining lines after that one.
  etlStatusSummary=$(echo "$etlStatusOut" | sed '1,/^}/d')

  etlStatusCombinedState=$(echo $etlStatusJson | jq -r .etlsCombinedStatus)

  if [[ "$etlStatusCombinedState" == "HEALTHY" ]] 
  then
    info "All ETLs are in a healthy status."
  else
    error "One or more ETLs are in an unhealthy status."
  fi

  msg "$etlStatusSummary\n"
  debug "$etlStatusJson\n"
}
```

I have a few "mixed" formats here, but in general internal variables should not be all uppercase in order to differentiate from environment variables and avoid potential conflicts. https://unix.stackexchange.com/a/42849